context/brand-context/package.json

```diff
@@ -4,5 +4,8 @@
   "license": "MIT",
   "description": "Bootstrapping for all components and products",
   "keywords": [],
-  "author": "Springer Nature"
+  "author": "Springer Nature",
+  "devDependencies": {
```

Comment:
I'm a bit confused, what is this for?

Reply:
The Jest tests fail if the @springernature imports in the tests and actual modules are not installed in the package